src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -235,19 +222,19 @@ fields for the maps.
 <pipeline_name>/
     sub-<participant_label>/
         dwi/
-            <source_keywords>[_space-<space>][_model-<label>][_desc-<label>]_<map_label>.nii[.gz]
+            <source_keywords>[_space-<space>][_desc-<label>]_model-<label>_<parameter>.nii[.gz]

```

Comment:
@Lestropie same comment here as about the use of 'parameter'
@jchoude parameter is more general, but we were trying to create a structure here, I believe, [_model + _desc + _map] 
This structure would allow human-friendly sorting in Unix, by model, as it was coming first.

The proposed structure [_desc + _model + _parameter] has a couple of issues for me:
(1) the term parameter is more open-ended and not as informative as _map
(2) we lose the sorting by 'model' by moving _desc to early.

Two cents.

Reply:
The sorting issue makes sense; I think my primary concern with the existing text was that "`_model`" was optional, not compulsory.

I think another thing I was going for here was that *intrinsic* parameters of the diffusion model would have "`[...]_model-<model>_parameter-<parameter>_diffmodel.nii[.gz]`", whereas *derived* parameters would be "`[...]_model-<model>_<parameter>.nii[.gz]`". The discussion around NODDI sort of highlights that the distinction between intrinsic and derived parameters is not clear-cut. Then again maybe using the same terminology in both locations is contributing to that, I don't know. I'm not a huge fan of "map", but having trouble articulating why; maybe it's my coding background, where a "map" provides a complex mapping from a dictionary to a set of values, or the fact that "map" in non-scientific parlance has a very different meaning.